src/ResourceManager/DataFactories/Commands.DataFactoryV2/help/Stop-AzureRmDataFactoryV2PipelineRun.md

```diff
@@ -0,0 +1,205 @@
+---
+external help file: Microsoft.Azure.Commands.DataFactoryV2.dll-Help.xml
+online version: 
```

Comment:
@hvermis for the two cmdlets you are adding, can you make sure that the `online version` is set to the following URL:

`https://docs.microsoft.com/en-us/powershell/module/azurerm.datafactories/<cmdlet_name>`

All existing cmdlet markdown files have this field, so feel free to use them as an example

Reply:
done